data/flexibleForms/forms.types.ts

```diff
@@ -27,8 +28,8 @@ export interface Field {
   className?: string;
   /** on conditional fields, required value is only respected when all conditions are met */
   required?: boolean;
-  /** give an initial, default value for string-type fields */
-  default?: string;
+  /** give an initial, default value for datetime and string-type fields */
+  default?: string | string[];
```

Comment:
Am i right in thinking the datetime would always be 2 strings, we could keep our types more restrictive using a tuple [string, string]

Reply:
makes sense